src/material/tree/testing/tree-harness.ts

```diff
@@ -6,10 +6,15 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {ComponentHarness, HarnessPredicate, parallel} from '@angular/cdk/testing';
 import {MatTreeNodeHarness} from './node-harness';
 import {TreeHarnessFilters, TreeNodeHarnessFilters} from './tree-harness-filters';
 
+type TreeNode = {
```

Comment:
This should be exported since it's used in the signature of a public API. Also, how about renaming it to `TextTree`